### PR TITLE
Added lint check on CI

### PR DESCRIPTION
**Pull request by sumitarora**
Should Fix: https://github.com/rangle/augury/issues/244

### CODE REVIEW

File: src/frontend/components/component-info/component-info.html [hunk 1]
```diff
@@ -1,91 +1,58 @@
 <div class="flex flex-column overflow-scroll col-12">
```
Comment (igor-ka) [inline src/frontend/components/component-info/component-info.html]: @sumitarora this is not linting related is it?

File: src/frontend/components/component-info/component-info.ts [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
 export default class ComponentInfo {
   private node: any;
+  private selectDependency: EventEmitter<string> = new EventEmitter<string>();
   private propertyTree: string = '';
-  private _input: Array<any>;
 
   constructor(
-    @Inject(ElementRef) private elementRef: ElementRef,
-    private userActions: UserActions
+    @Inject(ElementRef) private elementRef: ElementRef
```
Comment (igor-ka) [inline src/frontend/components/component-info/component-info.ts]: @sumitarora do we need the @Inject here? I would think that injection is happening based on type here anyway.

Comment (sumitarora) [inline src/frontend/components/component-info/component-info.ts]: @igor-ka no we don't I think this PR have some extra code removing that.

Comment (igor-ka) [inline src/frontend/components/component-info/component-info.ts]: 👍 I can't fin the extra code, but this is minor and we can fix it later.

**Comment by sumitarora** [comment]
@igor-ka done fixed conflicts

**Comment by igor-ka** [comment]
@sumitarora LGTM :shipit:

**Pull request merged**